lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -53,8 +53,8 @@ class Form526ToLighthouseTransform # rubocop:disable Metrics/ClassLength
 
       HAZARDS = {
         asbestos: 'Asbestos',
-        chemical: 'SHAD (Shipboard Hazard and Defense)',
-        mos: 'Military Occupational Specialty (MOS)-related toxin',
+        chemical: 'SHAD - Shipboard Hazard and Defense',
+        mos: 'Military Occupational Specialty-related toxin',
```

Comment:
Interesting that this hazard is not title-cased like all the others. Is that intentional?

Reply:
Also, not part of this PR but does `other: 'OTHER'` in `HAZARDS_LH_ENUM` (L71) correspond to `none: 'None of these')` for `HAZARDS` (L61)?

Reply:
> Interesting that this hazard is not title-cased like all the others. Is that intentional?
No, although it was signed off on this way 😂. I fixed it so it's at least consistent within this enum.

Reply:
> Also, not part of this PR but does other: 'OTHER' in HAZARDS_LH_ENUM (L71) correspond to none: 'None of these') for HAZARDS (L61)?
No. Each of those options have their own specialized handling. `none` means the user checked 'None of these' and is used in this and gulf war + hazard enums as well. `other` is a specialized option for hazards which is set only when the freeform text field for "Other (specify):" has value, and is sent in order to check the OTHER box on the digital form